src/main/java/com/deque/axe/AXE.java

```diff
@@ -223,16 +235,27 @@ public static void writeResults(final String name, final Object output) {
 		private final List<String> includes = new ArrayList<String>();
 		private final List<String> excludes = new ArrayList<String>();
 		private String options = "null";
+		private Boolean skipFrames = false;
 
 		/**
-		 * Injects the aXe script into the WebDriver.
+		 * Injects the aXe script into the WebDriver top document and all iframes.
 		 * @param driver An initialized WebDriver
+		 * @param script The javascript URL of aXe
 		 */
 		public Builder(final WebDriver driver, final URL script) {
 			this.driver = driver;
 			this.script = script;
 
-			AXE.inject(this.driver, this.script);
+			AXE.inject(this.driver, this.script, this.skipFrames);
```

Comment:
This line will have to move into the analyze function in order for the skipFrames builder function to take effect